app/controllers/employee_positions_controller.rb

```diff
@@ -0,0 +1,36 @@
+class EmployeePositionsController < ApplicationController
+  before_action :set_employee
+
+  def new
+    @employee_position = @employee.employee_positions.new
+  end
+
+  def create
+    position = Position.find_or_initialize_by(id: employee_position_params[:position_id])
+
+  if position.new_record?
```

Comment:
remove the possibility of creating one record from another controller. Especially with the hardcoded position name.